endoreg_db/serializers/report/report_list.py

```
@@ -0,0 +1,41 @@
+
+
+class ReportListSerializer(serializers.ModelSerializer):
+    """
+    Vereinfachter Serializer für Report-Listen
+    """
+    report_meta = ReportMetaSerializer(source='sensitive_meta', read_only=True)
+    file_type = serializers.SerializerMethodField()
+    status = serializers.SerializerMethodField()
+    updated_at = serializers.SerializerMethodField()
+
+    class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)